docs/aserto-console/policies.mdx

```diff
@@ -4,44 +4,125 @@ sidebar_label: Managing Policies
 
 # Managing Policies
 
-The underpinning of Aserto's authorization model is a [policy](/docs/overview/policy). 
+The underpinning of Aserto's authorization model is the [policy](/docs/overview/policy).
 
-Policies are authored, stored, and versioned as code in a `git` repository.
+**Policies** are authored, stored, and versioned as code in a `git` repository.
+
+A **policy instance** is an instance of the decision engine's runtime associated with a tagged version of a policy (or a tag mask). You can create multiple policy instances, each associated with a different version of a policy.
+
+To create a policy instance, you must first create a policy repository. Once the policy repository is set up, you can create a policy instance and associate it with a tag (or a tag mask) in the policy repository.
```

Comment:
tag masks are gone. when you bind to a tag, you're implicitly binding to the semver meaning of that tag.

Binding to "1" will bind to the latest semver version of the "1" lineage - e.g. "1.17.23".  Binding to "1.0" will bind to "1.0.27" if that 's the latest tag for that semver. Binding to "2.3.4" will bind directly to that tag.  

Binding to latest will implicitly bind to the latest tag (eg 3.4.5).

Reply:
(this isn't a verbatim recommendation - just explaining how the tags work now)